scripts/install_gh_wsl.sh

```
@@ -0,0 +1,30 @@
+  | dd of=/usr/share/keyrings/githubcli-archive-keyring.gpg status=none
+chmod go+r /usr/share/keyrings/githubcli-archive-keyring.gpg
+
+arch="$(dpkg --print-architecture)"
+dist="stable"
+echo "deb [arch=${arch} signed-by=/usr/share/keyrings/githubcli-archive-keyring.gpg] https://cli.github.com/packages ${dist} main" \
+  > /etc/apt/sources.list.d/github-cli.list
+
+apt-get update -y
+apt-get install -y gh
+
+echo "Installed:" $(gh --version | head -n1)
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)